### PR TITLE
[JBTM-3758] Missing test for Short Running Actions (SRA) on Quarkus

### PR DESCRIPTION
**Pull request by moulalis**
Issue: https://issues.redhat.com/browse/JBTM-3758

CORE TOMCAT AS_TESTS RTS JACOCO XTS QA_JTA QA_JTS_OPENJDKORB PERFORMANCE LRA DB_TESTS mysql db2 postgres oracle JDK17 JACOCO

### CODE REVIEW

File: rts/at/tx/pom.xml [hunk 1]
```diff
@@ -38,9 +38,18 @@
     <ssl.args>-Drts.usessl=true -Djavax.net.ssl.trustStore=keystore.jks -Djavax.net.ssl.trustStorePassword=123456</ssl.args>
     <!-- jakarta.servlet-api version 6.0.0 does not include jakarta/servlet/SingleThreadModel needed for undertow-->
     <version.jakarta.servlet.jakarta-servlet-api>5.0.0</version.jakarta.servlet.jakarta-servlet-api>
-
   </properties>
-
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
```
Comment (marcosgopen) [inline rts/at/tx/pom.xml:44]: These dependency changes impacted on LRA test https://ci-jenkins-csb-narayana.apps.ocp-c1.prod.psi.redhat.com/job/btny-pulls-narayana/PROFILE=LRA,jdk=openJDK17,label=jnlp-agent/82/consoleFull. I will have a look at it.

File: pom.xml [hunk 2]
```diff
@@ -269,7 +269,7 @@
     <version.org.springframework>3.2.2.RELEASE</version.org.springframework>
     <version.org.wildfly.arquillian>5.0.0.Alpha5</version.org.wildfly.arquillian>
     <version.postgresql>42.3.1</version.postgresql>
-    <version.quarkus>3.0.0.Alpha4</version.quarkus>
+    <version.quarkus>3.0.0.Alpha3</version.quarkus>
```
Comment (marcosgopen) [inline pom.xml:272]: Could you explain why we need to downgrade that dependency?

File: rts/at/tx/pom.xml [hunk 3]
```diff
@@ -38,9 +38,18 @@
...
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-bom</artifactId>
```
Comment (marcosgopen) [inline rts/at/tx/pom.xml:46]: Even if I usually like the use of bom (i.e in integration tests and quickstarts) I would prefer not to have the quarkus-bom in our lra api module. Let's see what @mmusgrov think about it.

File: rts/at/tx/pom.xml [hunk 4]
```diff
@@ -157,6 +166,15 @@
       <artifactId>jakarta.transaction-api</artifactId>
       <scope>provided</scope>
     </dependency>
+    <!-- Quarkus -->
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-resteasy</artifactId>
+    </dependency>
```
Comment (marcosgopen) [inline rts/at/tx/pom.xml:173]: I think quarkus-undertow is already included in quarkus-resteasy, so the quarkus-undertow dependency in redundant.

**Review by marcosgopen — CHANGES_REQUESTED** [review]
Thanks @moulalis for your PR!

File: rts/at/tx/pom.xml (hunk 1 above)
Comment (marcosgopen) [reply rts/at/tx/pom.xml:44]: I am sorry, those LRA error are not related to your PR. (see https://github.com/jbosstm/narayana/pull/2114).

File: pom.xml (hunk 2 above)
Comment (moulalis) [reply pom.xml:272]: @marcosgopen  Alpha4 is not compatible dependency which is throwing an error. It was just defined and not actively used in narayana. I checked with Alpha5 and it is working fine with it.

File: rts/at/tx/pom.xml (hunk 3 above)
Comment (moulalis) [reply rts/at/tx/pom.xml:46]: I think now I have used only one dependency i.e quarkus-resteasy. SO there is no sense of using quarkus-bom.

**Review by marcosgopen — APPROVED** [review]
Thank you @moulalis for your PR!

**Pull request merged**